packer/grapl-service/image.pkr.hcl

```diff
@@ -0,0 +1,163 @@
+packer {
+  required_version = ">= 0.12.0"
+}
+
+variable "build_ami" {
+  description = "Whether or not to actually build an AMI. Set to `false` for doing testing"
+  type        = bool
+  default     = true
+}
+
+variable "ami_name_prefix" {
+  type    = string
+  default = "grapl-service"
+}
+
+variable "aws_region" {
+  type    = string
+  default = "us-east-1"
+}
+
+variable "consul_module_version" {
+  type    = string
+  default = "v0.10.1"
+}
+
+variable "consul_version" {
+  type    = string
+  default = "1.9.6"
+}
+
+variable "nomad_version" {
+  type    = string
+  default = "1.1.1"
+}
+
+variable "terraform_aws_nomad_version" {
+  type    = string
```

Comment:
For instance, it would be useful to explain that this is actually a tag from the official github repository.

Actually renaming it to "terraform_aws_nomad_tag" would be useful too, particularly given the double-take someone might do when they see the `git clone --branch` command below.

Reply:
done